app/src/main/res/values-ja/strings.xml

```diff
@@ -97,5 +97,6 @@
     <string name="session_feedback_submit_failure">送信に失敗しました。</string>
     <string name="session_feedback_submit_success">送信しました。</string>
     <string name="session_feedback_submitlink">フィードバックを送信</string>
+    <string name="about_github_repository">GitHubリポジトリ</string>
```

Comment:
Sorry. I edited it by Translations Editor, so I did not notice it...
I'll fix it.